src/utils/customMessages.js

```diff
@@ -11,7 +11,15 @@ const messages = {
   notExistUser: 'User does not exist',
   changed: 'The password has been changed',
   notChanged: 'The password not changed',
-  errorMessage: 'occured an error'
+  errorMessage: 'occured an error',
+  alreadyExistEmail: 'user with the provided email exist',
+  invalidFirstname: 'firstName must consists of alphabets only and no whitespace or special characters',
```

Comment:
Please change this error to this and do it for the last name.
The first name must consist of alphabets only, no whitespaces or special characters.